kahuna/public/js/search/results.html

```diff
@@ -30,7 +30,7 @@
                         gr-tooltip-updates>
                     {{ctrl.newImagesCount | toLocaleString}} new
                 </button>
-                <button ng-if="ctrl.orgOwnedCount !== undefined && ctrl.orgOwnedCount !== ctrl.totalResults"
+                <button ng-if="ctrl.orgOwnedCount !== undefined && ctrl.orgOwnedCount !== ctrl.totalResults && (ctrl.orgOwnedCount + (ctrl.newOrgOwnedCount || 0))"
```

Comment:
nit: Is the condition implicitly `(ctrl.orgOwnedCount + (ctrl.newOrgOwnedCount || 0)) > 0`? it might be worth writing it out long-form just to avoid confusion
alternatively, this condition's getting fairly long - worth moving to controller and giving it a name?